core/schema_utils.py

```diff
@@ -723,3 +739,66 @@ def is_valid_username_string(obj: str) -> bool:
             return True
         except utils.ValidationError:
             return False
+
+    @staticmethod
+    def has_expected_subtitled_content_length(
+        obj: objects.SubtitledUnicode, max_value: int
+    ) -> bool:
+        """Checks if the given subtitled content length is within max value.
+
+        Args:
+            obj: objects.SubtitledUnicode. The object to verify.
+            max_value: int. The maximum allowed value for the obj.
+
+        Returns:
+            bool. Whether the given object has length atmost the max_value.
+        """
+        # Ruling out the possibility of different types for mypy type checking.
+        assert isinstance(obj, dict)
+        if len(obj['unicode_str']) > max_value:
+            return False
+        return True
+
+    @staticmethod
+    def has_subtitled_html_non_empty(
+        obj: objects.SubtitledHtml, is_default: bool = False
+    ) -> bool:
+        """Checks if the given subtitled html content is empty
+
+        Args:
+            obj: objects.SubtitledHtml. The object to verify.
+            is_default: bool. True when the incoming value is a `default_value`.
+
+        Returns:
+            bool. Whether the given object is empty.
+        """
+        # Ruling out the possibility of different types for mypy type checking.
+        assert isinstance(obj, dict)
+        if is_default:
+            return True
+        if obj['html'] in ('', '<p></p>'):
+            return False
+        return True
+
+    @staticmethod
+    def has_subtitled_html_content_uniquified(
```

Comment:
`has_unique_subtitled_contents`

Reply:
(not html in the name because we can make it work for all the subtitled contents [HTML, unicode etc.] in future if needed)